apps/gallery/style/gallery.css

```diff
@@ -920,3 +920,31 @@ a.filter-faded {
 #slider3 {
   left: calc(3.9rem + (100% - 2 * 3.9rem));
 }
+
+html[dir="rtl"] #slider-3 {
+  left: calc(3.9rem + (100% - 2 * 3.9rem));
+}
+
+html[dir="rtl"] #slider-2 {
+  left: calc(3.9rem + (100% - 2 * 3.9rem) * 5 / 6);
+}
+
+html[dir="rtl"] #slider-1 {
+  left: calc(3.9rem + (100% - 2 * 3.9rem) * 4 / 6);
+}
+
+html[dir="rtl"] #slider0{
+  left: calc(3.9rem + (100% - 2 * 3.9rem) * 3 / 6);
+}
+
+html[dir="rtl"] #slider1 {
+  left: calc(3.9rem + (100% - 2 * 3.9rem) * 2 / 6);
+}
+
+html[dir="rtl"] #slider2 {
+  left: calc(3.9rem + (100% - 2 * 3.9rem) / 6);
+}
+
+html[dir="rtl"] #slider3 {
+  left: 3.9rem;
+}
```

Comment:
I wish there was a way to make this all less verbose. Would the RTL ones work if you set right instead of left? That would make them more parallel with the LTR styles because they'd have the same calculations.

Or, can you make it work by settting left and right to 0, and then setting margin-start to the calculated value? If that works, then you wouldn't need the special case for rtl.

This seems okay as it is, and I don't want to ask you to do a lot more work, so if margin-start can work, I'd suggest using that, but otherwise this is fine.